concert/experiments/addons.py

```diff
@@ -398,9 +399,11 @@ async def _check_timestamp(self, producer):
                                                "Works only with pco cameras.")
                     self.timestamp_missing = True
                     return
-            if img.metadata['frame_number'] != i + 1:
+            if i == 0:
+                first_frame = img.metadata['frame_number']
+            if img.metadata['frame_number'] != i + first_frame:
```

Comment:
Is it safe to rely on the first frame index being 1? All PCO's behave the same way for sure?